modules/ROOT/pages/archives.adoc

```diff
@@ -3,12 +3,12 @@
 
 This page gives access to the download of Bonita documentation archives, in case you need something very specific to those versions. The format is either html (for not so old versions) or pdf (for very old versions).
 
-== Bonita BPM 7.3 to Bonita BPM 7.5
+== Bonita BPM 7.3 to Bonita BPM 7.6
 
 The format here is html. +
 Download the .zip archive, extract it, and then open the index.html file to launch the site. +
 You can navigate those sites with the table of content on the left. 
-
+* https://github.com/bonitasoft/bonita-doc/releases/download/7.6-20220330_125930/documentation-bonita-7.6.zip[Bonita BPM 7.6]
```

Comment:
⚠️ in v7.6, Bonita BPM was renamed into Bonita.
@NathalieC do we create a new paragraph for Bonita?